plugins/example-code-snippets/src/api-support/api-support-matrix-generator.ts

```diff
@@ -53,7 +53,7 @@ const SDKS: Array<SdkInfo> = [
     cacheClientFile: 'src/Momento.Sdk/ICacheClient.cs',
     configObjectFile: 'src/Momento.Sdk/Config/Configuration.cs',
     topicClientFile: 'src/Momento.Sdk/ITopicClient.cs',
-    authClientFile: undefined,
+    authClientFile: 'src/Momento.Sdk/IAuthClient.cs',
```

Comment:
generateDisposableToken was implemented in dotnet sdk but it wasn't showing up in the language support matrix until this file was added